src/components/StatsTooltip/StatsTooltipRow.tsx

```diff
@@ -1,7 +1,7 @@
 import "./StatsTooltip.css";
 type Props = {
   label: string;
-  value: number | [number];
+  value: number | string | [string] | [number];
```

Comment:
The `[string]` type means "Array with exactly one string element".

If you want to say "Array of strings," you must use the type `string[]`

<img width="1291" alt="Screenshot 2022-11-04 at 19 38 43" src="https://user-images.githubusercontent.com/110596516/200016150-fad6977d-25e8-42da-9ab9-a865c6fef42f.png">

Reply:
Resolved! Sorry silly mistake 🥲